stellar/its-example.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { Contract, Address, nativeToScVal } = require('@stellar/stellar-sdk');
+const { Command } = require('commander');
+const { loadConfig, saveConfig } = require('../evm/utils');
+const { getWallet, broadcast, addBaseOptions, tokenToScVal, tokenMetadataToScVal } = require('./utils');
+const { addOptionsToCommands, getChainConfig } = require('../common');
+const { ethers } = require('hardhat');
+const {
+    utils: { arrayify, hexZeroPad },
+} = ethers;
+require('./cli-utils');
+
+async function deployToken(wallet, _, chain, contractConfig, args, options) {
+    const contract = new Contract(contractConfig.address);
+    const caller = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
+    const minter = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
```

Comment:
```suggestion
    const caller = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
    const minter = caller;
```
Or better yet, take in an optional minter. If it's equal to `'wallet'` then reuse the wallet, or if it's undefined, use None

Reply:
updated to use `const minter = caller;` for now.